spec/truemail/validate/smtp/request_spec.rb

```diff
@@ -42,16 +42,15 @@
 
     context 'when port opened' do
       specify do
-        allow(Timeout).to receive(:timeout).with(connection_timeout).and_call_original
-        allow(TCPSocket).to receive_message_chain(:new, :close)
+        allow(Socket).to receive(:tcp).and_return(true)
```

Comment:
For case when we use stubs, you should check all passed params, blocks, etc... Please use code snippet from example above.